JSDemos/Demos/Charts/LoadDataOnDemand/description.md

```diff
@@ -1 +1,29 @@
-In&nbsp;this demo, the chart loads data as&nbsp;you pan&nbsp;it. Once a&nbsp;data block is&nbsp;loaded, it&nbsp;stays in&nbsp;memory to&nbsp;reduce requests to&nbsp;the server.
\ No newline at end of file
+In this demo, the [range area](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/RangeArea/) Chart loads data as you pan it. Once the component loads a data block, this data stays in memory to reduce requests to the server. To implement this functionality, do the following:
+
+1. Configure a data source.
+
+    - Assign a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) with an empty [store](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/store/) to the Chart [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource) property.
+
+    - Set the [sort](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#sort) property to `date`.
+
+    - Disable the [paginate](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#paginate) property to prevent data from partitioning.
+
+2. Configure the Chart to support the data load on demand.
+
+    - Set the [visualRangeUpdateMode](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#visualRangeUpdateMode) property to `keep`.
+
+    - Specify a [loadingIndicator](/Documentation/ApiReference/UI_Components/dxChart/Configuration/loadingIndicator/).
+
+    - Specify the initial [visualRange](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/visualRange/).
+    
+    - Implement a function that computes the Chart current [visualRange](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/visualRange/). In this function, call a method that checks whether a user changed the visual range.
```

Comment:
As I understand, here we handle the control's event that fires when a user pans the chart. If the range changed, we initiate data loading operations. 

```suggestion
    - Implement a function that respond to user pan operations and tracks [visualRange](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/visualRange/) changes. If a user changed the visual range, initiate a request for additional data.
```